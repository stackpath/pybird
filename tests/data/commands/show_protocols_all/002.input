
0001 BIRD 2.0.4 ready.
2002-Name       Proto      Table      State  Since         Info
1002-bfd1       BFD        ---        up     2019-06-03
1006-
1002-peer_aaa10 BGP        ---        start  00:42:26.125  Active        Socket: Connection refused
1006-  Description:    Peer session - aaa10
   BGP state:          Active
     Neighbor address: 192.168.8.1
     Neighbor AS:      4220101000
     Connect delay:    0.142/5
     Last error:       Socket: Connection refused
   Channel ipv4
     State:          DOWN
     Table:          foo_net
     Preference:     100
     Input filter:   (unnamed)
     Output filter:  (unnamed)
     Import limit:   25000
       Action:       disable
     IGP IPv4 table: foo_net

0000