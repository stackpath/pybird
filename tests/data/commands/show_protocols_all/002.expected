[
    {"protocol": "BGP", 
    "description": "Peer session - aaa10", 
    "ipv4_state": "DOWN",
    "last_error": "Socket: Connection refused",
    "up": false, 
    "state": "Active", 
    "address": "192.168.8.1", 
    "last_change": "2019-06-05T00:42:00", 
    "asn": "4220101000", 
    "name": "peer_aaa10"}
]